labelbox/schema/project.py

```diff
@@ -773,6 +779,156 @@ def create_batch(
             return self._create_batch_sync(name, dr_ids, global_keys, priority,
                                            consensus_settings)
 
+    def create_batches(
+        self,
+        name_prefix: str,
+        data_rows: Optional[List[Union[str, DataRow]]] = None,
```

Comment:
is the number of data rows 'infinite'? i.e. do we know whether it will blow up after certain large number of rows?